Gemfile.lock

```diff
@@ -1,233 +1,242 @@
 GEM
```

Comment:
Annoyingly, I had to change a few dependencies in the Gemfile to get the framework working on my mac! I can remove these changes once approved for merging!